vtex/react-jsonschema-table
# Refactor

This refactor features:
 - Receive items from props, so fetching can be controlled outside component.
 - Redux actions and reducers refactored.
 - New Folder structure.
 - Loads of minor enhancements in function scopes and names.
 - Enhanced documentation of component API. (a little)
 - Make more examples.
 - Fix #16 
 - Fix #14 
 - Fix #12 
 - Fix #6 
 - Fix #8 
 - Partially Fix #11 and #7 

How to test it:
 - Check out Profile System admin (https://baguera--ambientecrm.myvtexdev.com/admin/profile-system)
 - Or run it locally with `npm start`
